### PR TITLE
PDE-4140 feat(cli): Integration title length must be at least 2 characters

### PR DESCRIPTION
**Pull request by pragmatic-zac**
This PR adds a minimum length constraint for an integration title when adding via zapier register.

### CODE REVIEW

File: packages/cli/src/oclif/commands/register.js [hunk 1]
```diff
@@ -241,7 +253,7 @@ RegisterCommand.args = [
   {
     name: 'title',
     description:
-      "Your integrations's public title. Asked interactively if not present.",
+      "Your integration's public title. Asked interactively if not present.",
```
Comment (rnegron) [inline packages/cli/src/oclif/commands/register.js:256]: Good catch!

File: packages/cli/src/oclif/commands/register.js [hunk 2]
```diff
@@ -28,6 +33,12 @@ class RegisterCommand extends ZapierBaseCommand {
       );
     }
 
+    if ('title' in this.args && this.args.title.length < MIN_TITLE_LENGTH) {
```
Comment (rnegron) [inline packages/cli/src/oclif/commands/register.js:36]: @pragmatic-zac When running `zapier register` (without an argument, thus triggering this flow), I am getting: `Error: Cannot read properties of undefined (reading 'length')`. Using the `--debug` flag shows that ` zapier:register args are { title: undefined }` so it seems that checking for just `'title' in this.args` is not enough 😢

**Review by rnegron — CHANGES_REQUESTED** [review]

File: packages/cli/src/oclif/commands/register.js (hunk 2 above)
Comment (pragmatic-zac) [reply packages/cli/src/oclif/commands/register.js]: That's a really helpful shell alias, thank you!

Made a slight adjustment and this is fixed in my manual tests now. Ready for another look whenever you're able 😄

**Comment by rnegron** [comment]
@pragmatic-zac I am seeing some failed CI tests on the latest commit. Tried re-running them but they failed again so they don't seem to be flaky tests.

**Pull request merged**